owncloudApp/src/main/res/values/strings.xml

```diff
@@ -617,11 +617,6 @@
     <string name="action_upload_retry">Retry</string>
     <string name="action_upload_clear">Clear</string>
 
-    <string name="manage_space_title">Manage space</string>
-    <string name="manage_space_description">This feature is not currently available. Next versions will include new features for automatic and manual removal of storage. Stay tuned!</string>
-    <string name="manage_space_clear_data">Clear data</string>
-    <string name="manage_space_error">Some files could not be deleted.</string>
```

Comment:
This is not enough, remember that when we remove a String, we need to remove it in every language...

Reply:
Sorry...done